java/tools/src/test/java/org/apache/arrow/tools/ArrowFileTestFixtures.java

```diff
@@ -94,4 +172,14 @@ static void writeInput(File testInFile, BufferAllocator allocator) throws IOExce
       write(parent.getChild("root"), testInFile);
     }
   }
+
+  static void writeVariableWidthViewInput(File testInFile, BufferAllocator allocator, int count)
```

Comment:
Where are these used?

Reply:
This is used in the `TestIntegration.java`

Reply:
@lidavidm any issues here?